modular/gater/admin_handler.go

```diff
@@ -46,7 +46,7 @@ func (g *GateModular) getApprovalHandler(w http.ResponseWriter, r *http.Request)
 		log.CtxDebugw(reqCtx.Context(), reqCtx.String())
 	}()
 
-	reqCtx, err = NewRequestContext(r)
+	reqCtx, err = g.NewRequestContext(r)
```

Comment:
NewRequestContext(r, g), let RequestContext combining resources of g is better?
RequestContext seems not belong to g from function view.

Reply:
Done